internal/serve/serve.go

```diff
@@ -126,6 +144,17 @@ func getHandlerDeps(cfg Configs) (handlerDeps, error) {
 	}, nil
 }
 
+func ensureChannelAccounts(channelAccountService services.ChannelAccountService, numberOfChannelAccounts int64) {
+	ctx := context.Background()
+	log.Ctx(ctx).Info("Ensuring the number of channel accounts in the database...")
+	err := channelAccountService.EnsureChannelAccounts(ctx, numberOfChannelAccounts)
+	if err != nil {
+		log.Ctx(ctx).Errorf("error ensuring the number of channel accounts: %s", err.Error())
+		return
```

Comment:
Whenever we implement Sentry, I think this might be worthy of a Sentry exception as well, wdyt?